rosa-hypershift/rosa-hosted-wrapper.py

```diff
@@ -660,7 +660,13 @@ def _build_cluster(ocm_cmnd, rosa_cmnd, cluster_name_seed, must_gather_all, prov
     logging.debug('Attempting cluster installation')
     logging.debug('Output directory set to %s' % cluster_path)
     cluster_name = cluster_name_seed + "-" + str(my_inc).zfill(4)
-    cluster_cmd = [rosa_cmnd, "create", "cluster", "--cluster-name", cluster_name, "--replicas", str(worker_nodes), "--hosted-cp", "--sts", "--mode", "auto", "-y", "--output", "json", "--oidc-config-id", oidc_config_id]
+    git_user = _get_git_repo_details()
+    if git_user != "cloud-bulldozer":
+        GITHUB_USERNAME = git_user
+    else:
+        GITHUB_USERNAME = "cloud-bulldozer"
+
+    cluster_cmd = [rosa_cmnd, "create", "cluster", "--cluster-name", cluster_name, f"--tags=User:{GITHUB_USERNAME}", "--replicas", str(worker_nodes), "--hosted-cp", "--sts", "--mode", "auto", "-y", "--output", "json", "--oidc-config-id", oidc_config_id]
```

Comment:
@krishvoor I honeslty don't think that the github user is the most useful tag to apply here. I'd rather we take in a string from the CLI to tag whatever we need it to be, ie some financial tracking, username, or whatever

Reply:
> @krishvoor I honeslty don't think that the github user is the most useful tag to apply here. 

@dry923 I hear you, today the [cloud expenses](https://github.com/redhat-performance/cloud-governance/blob/main/cloud_governance/policy/policy_operations/aws/tag_cluster/tag_cluster_operations.py#L74) are tracked via GitHub user details just like the one defined here [via ROSA DAGs](https://github.com/cloud-bulldozer/airflow-kubernetes/blob/master/dags/openshift_nightlies/scripts/install/rosa.sh#L430) and have proved useful.

> I'd rather we take in a string from the CLI to tag whatever we need it to be, ie some financial tracking, username, or whatever

Agreed, will implement changes through with this as an additional change.